redis-express/README.md

```diff
@@ -0,0 +1,164 @@
+# Express Redis Sample
+
+## Overview
+
+This API manages books record, built with Express and Redis. It provides routes to add, update, delete, view books, view books by ID, and change task authors.
+
+
+## Installation
+
+### Prerequisites
+
+Ensure you have the following installed:
+
+- Docker
+- Node.js and npm
+- Keploy CLI
+# Installing Keploy
+
+Let's get started by setting up the Keploy alias with this command:
+
+```sh
+curl -O https://raw.githubusercontent.com/keploy/keploy/main/keploy.sh && source keploy.sh
+```
+
+Clone the repository and move to express-redis folder
+
+```bash
+git clone https://github.com/keploy/samples-typescript.git
+cd samples-typescript/express-redis
+```
+
+
+### Install the dependencies
+```bash
+npm install
+```
+
+
+### Start Redis-server Container
+```bash
+docker run -d --name redis-stack-server -p 6379:6379 redis/redis-stack-server:latest
+```
+
+
+
+### Load the Samples and build server:
+
+
+In case of running application natively, we will have to change the host to localhost in index.ts
+
+
```

Comment:
Please check the Preview of this code, looks disoriented 
<img width="937" alt="Screenshot 2024-10-01 at 18 34 05" src="https://github.com/user-attachments/assets/80d68714-6b04-47d1-a6d5-d4e8f730fdcc">